Header header
float32[] position
float32 yaw
float32[] velocity
float32 yawVelocity